NorimaConsulting/training-and-onboarding
# REC-32: Basic Nav Structure

## Resolves/Added: 

- Basic User Navigation between different pages/routes

### Testing:

1. Go to: http://localhost:3000
2. In terminal run: *pnpm dev*
3. Click on any links on the NavBar (some just route back to "/" until that component is made)
4. Click on the user icon in NavBar. A Drop-Down menu should appear, and when you click on _Profile_ it should take you to the users profile. 
5. When an individual recipe card is clicked it should route to the "/recipe" showing a single recipe page. 

#### Screen Shots of what it should look like: 

**Home Page:** 
<img width="752" alt="image" src="https://user-images.githubusercontent.com/83307690/149568575-4ec701d5-0f1a-41de-91b6-76fa1ba7ffc1.png">

**List of Recipes:** 
<img width="751" alt="image" src="https://user-images.githubusercontent.com/83307690/149568632-310c5cca-1cb5-4df3-8cf0-4c9b6bec8fa0.png">

**User Profile **
<img width="762" alt="image" src="https://user-images.githubusercontent.com/83307690/149568680-6025ee13-6ccc-4f5b-b59c-67283e70bad0.png">
